ART Plugins,TKY Plugins and ART SUB APP Fan Identity

alexanderejm opened the pull request:
REFACTOR: Add License to all plugins in ART and TKY, Include in: #6100, #6099
FIX: Validation fixes: fixed an error that prevented the identity to update the externalPlatformID

alexanderejm committed:
REFACTOR: Add License to all plugins in ART and TKY

Include in: #6100, #6099

alexanderejm committed:
FIX: Validation fixes

fixed an error that prevented the identity to update the externalPlatformID

alexanderejm committed:
Merge branch 'develop' of https://github.com/darkestpriest/fermat into develop

darkestpriest commented:
This pull request has conflicts, please @alexanderejm resolve it and update your repository.

alexanderejm committed:
REFACTOR/FIX: fix 2 bugs and refactor spinner actionListener

Fix Bug that allows to change platform while maintaining externalID from another platform
 Fix NullPointer in "identitySelected.getExternalIdentityID()" when ExternalIdentityID is null

 Refactor ExternalUser SpinnerActionListener content to createNewIdentity to erase global externalIdentityID variable

Included in #6105, #6107

alexanderejm committed:
Merge branch 'develop' of https://github.com/darkestpriest/fermat into develop

Conflicts:
	ART/android/sub_app/fermat-art-android-sub-app-fan-identity-bitdubai/src/main/java/com/bitdubai/sub_app/art_fan_identity/fragments/CreateArtFanUserIdentityFragment.java
